crates/turborepo-lib/src/commands/query.rs

```diff
@@ -58,10 +59,20 @@ pub async fn run(
     mut base: CommandBase,
     telemetry: CommandEventBuilder,
     query: Option<String>,
+    variables_path: Option<&Utf8Path>,
 ) -> Result<i32, Error> {
     let signal = get_signal()?;
     let handler = SignalHandler::new(signal);
 
+    let variables: Variables = variables_path
```

Comment:
suggestion: (sorry if I missed a good reason not to do this) might be nice to move this down with where it's used in the hunk below this one